### PR TITLE
Azure EventHubs test fixup

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: integration-tests/azure-eventhubs/azure-resources.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+suffix="$(az ad signed-in-user show --query displayName -o tsv | tr '[:upper:]' '[:lower:]' | tr -cd '[:alnum:]' | cut -c-12)"
+suffix="${suffix}1"
+export AZURE_STORAGE_ACCOUNT_NAME=cqacc${suffix}
+export AZURE_BLOB_CONTAINER_NAME=cq-container-${suffix}
+
+export RESOURCE_GROUP=cq-res-group-${suffix}
+export ZONE=westeurope
+export EH_NAMESPACE=cq-eh-namenspace-${suffix}
```
Comment (llowinge) [inline integration-tests/azure-eventhubs/azure-resources.sh:27]: @ppalaga Typo in `namenspace`

**Comment by llowinge** [comment]
@ppalaga I've tried the script and it failed with `Insufficient privileges to complete the operation.` after `az role assignment create`

**Comment by ppalaga** [comment]
> @ppalaga I've tried the script and it failed with `Insufficient privileges to complete the operation.` after `az role assignment create`

Maybe because you are not the account owner? I am the owner and it works for me.

**Comment by llowinge** [comment]
@ppalaga Yes, probably. It makes the script unusable for me :( However the fix in the test is working with my previously created credentials. Thanks a lot !

**Pull request merged**